python/ray/experimental/slib/streaming.py

```diff
@@ -0,0 +1,339 @@
+import sys
+import uuid
+import networkx as nx
```

Comment:
This is a third-party library, right? Add to the `README.rst` installation instructions